remote/mocks/Remote.go

```diff
@@ -1,11 +1,15 @@
-// Code generated by mockery v1.0.0. DO NOT EDIT.
+// Code generated by mockery v0.0.0-dev. DO NOT EDIT.
 
```

Comment:
this is unrelated please move it into it's own pull!!

Reply:
That was some incorrect merge. I reverted it.